cxx-qt-build/src/lib.rs

```diff
@@ -268,43 +277,112 @@ impl CxxQtBuilder {
         self
     }
 
-    /// Choose to disable Qt support
-    ///
-    /// This will disable including cxx-qt-lib headers.
-    pub fn disable_qt(mut self) -> Self {
-        self.qt_enabled = false;
-        self
-    }
-
     /// Specify rust file paths to parse through the cxx-qt marco
     ///
     /// Currently the path should be relative to CARGO_MANIFEST_DIR
     pub fn file(mut self, rust_source: &'static str) -> Self {
         self.rust_sources.push(rust_source);
+        println!("cargo:rerun-if-changed={}", rust_source);
         self
     }
 
-    // TODO: support globs with files("src/**/*.rs")
+    /// Link additional [Qt modules](https://doc.qt.io/qt-6/qtmodules.html).
+    /// Specify their names without the `Qt` prefix, for example `"Widgets"`.
+    /// The Core and Gui modules are linked automatically; there is no need to specify them.
+    pub fn qt_modules(mut self, modules: &[&str]) -> Self {
+        for module in modules {
```

Comment:
seems this can also be

```rust
        self.qt_modules.extend(modules.iter().cloned().map(String::from));
```
not sure if it's better or worse :-)

Reply:
sure